gulpfile.js

```diff
@@ -121,7 +122,7 @@ gulp.task('compile-tests', function () {
 
 gulp.task('default', function () {
     runSequence(
-        'watch',
-        $.shell.task('PATH=node_modules/.bin:$PATH electron .')
+        'clean',
+        ['typescript', 'sass', 'react']
```

Comment:
This is new usage for default task that makes watch task unused, should we delete it then?